modules/programs/gpg.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.gpg;
+
+  cfgText = concatStringsSep "\n" (attrValues (mapAttrs (key: value:
+    if isString value then "${key} ${value}"
+    else optionalString value key) cfg.configuration
+  ));
+
+in {
+  options.programs.gpg = {
+    enable = mkEnableOption "gnupg";
+
+    configuration = mkOption {
+      description = ''
+        GPG configuration options. Avalible options are described in gpg manpage:
```

Comment:
Just because I stumbled over this, you may appreciate fixing the little typo in: Available (currently Avalible) in the doc string :)